support/cas-server-support-memcached-ticket-registry/src/main/java/org/apereo/cas/ticket/registry/support/kryo/KryoTranscoder.java

```diff
@@ -111,10 +123,17 @@ public void initialize() {
         this.kryo.register(HashMap.class);
         this.kryo.register(LinkedHashMap.class);
         this.kryo.register(HashSet.class);
+        Set singletonSet = Collections.singleton("test");
```

Comment:
I'd be happy to redo this bit - but the SingletonSet and SingletonMap classes in Collections are private - to register those all I could think of was to instantiate one of each, and do a getClass() on the results...

Reply:
p.s. - the warning for unregistered classes works great - it made it easy to figure out what additional classes needed to be registered when I doubled up on the deserializations in the test code :)

Reply:
All good. Much appreciated. Who could say no to more tests :)

You just need to fix the styling issues I believe; if checkstyle complains about your use of `singleton()` and such, simply add a rule to suppress those errors and make an exception.